plugins.json

```diff
@@ -1,5 +1,4 @@
 {
```

Comment:
Should this be updated to
```json
[
  "auto-sizes",
  "dominant-color-images",
  "speculation-rules",
  "webp-uploads"
]
```
Since `plugins.json` only store plugin names anyway.